web/src/containers/activity/Goals.js

```diff
@@ -1,61 +1,133 @@
+import { Button, Review, TextField } from '@cmsgov/design-system-core';
+
 import PropTypes from 'prop-types';
-import React, { Component, Fragment } from 'react';
+import React, { Component, Fragment, useState } from 'react';
 import { connect } from 'react-redux';
 
 import {
   addActivityGoal as addActivityGoalAction,
   removeActivityGoal as removeActivityGoalAction,
   updateActivity as updateActivityAction
 } from '../../actions/activities';
-import Btn from '../../components/Btn';
-import CollapsibleList from '../../components/CollapsibleList';
-import { RichText } from '../../components/Inputs';
-import Instruction from '../../components/Instruction';
 import NoDataMsg from '../../components/NoDataMsg';
 import { Subsection } from '../../components/Section';
 import { t } from '../../i18n';
 
-const plaintext = txt => {
-  const e = document.createElement('div');
-  e.innerHTML = txt;
-  return e.innerText;
+const GoalReview = ({ goal, idx, edit, handleDelete }) => (
+  <Review
+    heading={
+      <h5 className="ds-h4">
+        {idx + 1}. {goal.description}
+      </h5>
+    }
+    editHref="#"
+    onEditClick={edit}
+    editContent={
+      <div className="nowrap">
+        <Button size="small" variation="transparent" onClick={edit}>
+          Edit
+        </Button>
+        {handleDelete && (
+          <Fragment>
+            |
+            <Button size="small" variation="transparent" onClick={handleDelete}>
+              Remove
+            </Button>
+          </Fragment>
+        )}
+      </div>
+    }
+  >
+    <p className="ds-u-margin-top--2">Benchmark: {goal.objective}</p>
+  </Review>
```

Comment:
This is the third time I've done this exact pattern and I'm thinking it'd make sense to create a custom wrapper around `<Review>` that does all the header stuff for us automatically, and also consistently...

Reply:
![yesplz](https://media3.giphy.com/media/3o85xvgnZafHvlQO8U/giphy.gif?cid=790b76115cf5a77a3576726d554d6406&rid=giphy.gif)